README.md

```diff
@@ -64,6 +64,70 @@ Follow me [![twitter](https://img.shields.io/twitter/follow/valorkin.svg?style=s
   }
 ```
 
+### With webpack and bootstrap-sass: [AngularClass/angular2-webpack-starter](https://github.com/AngularClass/angular2-webpack-starter)
+- Install `ng2-bootstrap` and loaders for webpack
+```bash
+  npm install --save-dev sass-loader css-to-string-loader  resolve-url-loader file-loader url-loader node-sass
+  npm install --save bootstrap-sass ng2-bootstrap
+```
+
+ - Add loaders to `webpack.common.js`:
+```javascript
+      /**config/webpack.common.js*/
+      {
+        test: /\.sass$/,
+        loaders: ['css-to-string-loader', 'css-loader', 'resolve-url', 'sass-loader?indentedSyntax=true&sourceMap']
+      },
+      {
+        test: /\.scss$/,
+        loaders: ['css-to-string-loader', 'css-loader', 'resolve-url', 'sass-loader?indentedSyntax=false&sourceMap']
+      },
+      { test: /\.woff(2)?(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: "url?limit=10000&minetype=application/font-woff" },
+      { test: /\.(ttf|eot|svg)(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: "file" },
+```
+ - Rename `src/app/app.style.scss` to `src/app/app.style.scss` and add the following line to `src/app/app.styles.scss`:
```

Comment:
Hey, I checked and updated pull request. In absence of indentedSyntax parameter node-sass is smart enough to figure out which syntax to use based on filename.